.github/workflows/hz-enterprise-op.-rhel-release.yaml

```diff
@@ -375,3 +376,20 @@ jobs:
           gh pr create \
               --title "Update versions after release of RHEL ${OPERATOR_IMAGE}" \
               --body "Released ${OPERATOR_IMAGE} on RHEL. New operator and operator-bundle images are released on RHEL registry. "
+
+
+  slack_notify:
+    name: Slack Notify
+    needs: build_publish 
+    runs-on: ubuntu-latest
+    if: always()
+    steps:
+      - uses: 8398a7/action-slack@v3
+        if: needs.build_publish.result != 'success'
+        with:
+          fields: repo,commit,author,action,eventName,workflow
+          status: ${{ needs.build_publish.result }}
+          channel: '#github-actions-log'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
In terms of security, do we have any policy what GH Actions we use and what credentials we can pass to them? I remember that it was a concern of @jerrinot some time ago.

CC: @kwart

Reply:
`GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}` This token has scope only for this repository and it is created default by Github.

Reply:
AFAIK we don't have such a policy yet.

What we should probably do is to create a list of trusted organizations (i.e. github accounts). Actions from these trusted accounts would be allowed to work with sensitive information.
If we want to use an action from a user/organization which is not listed and the action would use a sensitive information, then we should fork the action code, review it and reference the action from our own repository.

Reply:
@mtyazici Could you please do the research? I mean to do the following:

1. Check what possible GH Actions we can use for the Slack Notifications.
2. Check their popularity / usage (how many stars, what other tools use them)
3. Ping Josef on slack and discuss if we should just use the given action or should we fork the repo

Reply:
Sure, I will